platform/spec/services/syncs/create_service_spec.rb

```
@@ -0,0 +1,37 @@
+      expect(syncs[0]).to have_attributes(
+        stream_name: "stream1",
+        sync_mode: "incremental_append",
+        supported_sync_modes: ["full_refresh", "incremental"]
+      )
+
+      expect(syncs[1]).to have_attributes(
+        stream_name: "stream2",
+        sync_mode: "full_refresh_overwrite",
+        supported_sync_modes: ["full_refresh"]
+      )
+    end
```

rubocop

[convention] Example has too many lines. [12/10]

(RSpec/ExampleLength)